docs/nativescript.md

```diff
@@ -0,0 +1,61 @@
+# Using Mimic with NativeScript
+
+Mimic allows you to see and mock all networks requests coming from your NativeScript application.
+In this guide we will see how we can easily install Mimic into our application and mock our first request.
+
+Check out the [example repository](https://github.com/morsdyce/mimic-nativescript-example).
+
+
+## Installation
+
+Since with NativeScript we don't have access to the debugger, in order to access the Mimic UI you would need both
+the mimic itself and the proxy that would connect the UI with your NativeScript application.
+
+```
+$ npm install -g mimic-remote
+$ npm install -D mimic
+```
+
+Since mimic uses websockets to communicate with its UI,
+you would also need to add the [nativescript-websockets](https://www.npmjs.com/package/nativescript-websockets) plugin:
+
+```
+$ npm install -S nativescript-websockets
+$ tns plugin add nativescript-websockets
+```
+
+* `-S` is a shortcut for --save
```

Comment:
I think it's better to always use the long flags rather than explaining them afterwards, it will be most understandable and users will have less data to consume since we want them to focus on mimic itself rather than other tools.